build/signing-config-mac.yaml

```diff
@@ -0,0 +1,59 @@
+# This file describes the code signing configuration for macOS.
+
+# List of entitlements.
+entitlements:
+  # This contains the default entitlements, for files not otherwise listed.
+  default:
+  - com.apple.security.inherit
+
+  # Entitlement overrides.  This is a list of overrides, each with a "paths"
+  # key describing which paths to override, and an "entitlements" key for the
+  # overriding entitlements.
+  overrides:
+  - paths:
+    - '' # This is the main application
+    entitlements:
+    - com.apple.security.cs.allow-jit
+    - com.apple.security.network.server
+    - com.apple.security.network.client
```

Comment:
I've checked why Lima has the `com.apple.security.network.server` and `com.apple.security.network.client` entitlements, and it turns out they were just copied from https://github.com/Code-Hex/vz/blob/bd29a7ea3d39465c4224bfb01e990e8c220a8449/example/linux/vz.entitlements, the library that Lima is using to implement VZ emulation.

I believe they are only needed for sandboxed App Store applications, so we can probably just remove them everywhere, to get a smaller set of entitlements.